components/Utilities/SharedBus/SharedBus.h

```diff
@@ -0,0 +1,45 @@
+#ifndef SHARED_BUS_H
+#define SHARED_BUS_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <freertos/FreeRTOS.h>
+#include <freertos/queue.h>
+#include <freertos/task.h>
+#include <freertos/event_groups.h>
+
+typedef enum {
```

Comment:
I feel this enum should be somewhere in `main.c` or `main.h`

Reply:
For some reason, I say NO. we're going to discuss it in the daily meeting.